src/main/java/org/craftercms/studio/api/v2/event/workflow/WorkflowEvent.java

```diff
@@ -29,22 +30,47 @@
  */
 public class WorkflowEvent extends SiteAwareEvent implements SiteBroadcastEvent {
 
-    public WorkflowEvent(Authentication authentication, String siteId) {
-        super(authentication, siteId);
+    private final WorkFlowEventType eventType;
+    private final long packageId;
+
+    public WorkflowEvent(final String siteId, final long packageId, final WorkFlowEventType eventType) {
+        super(siteId);
+        this.eventType = eventType;
+        this.packageId = packageId;
     }
 
     @Override
     public String getEventType() {
-        return "WORKFLOW_EVENT";
+        return format("WORKFLOW_EVENT_%s", eventType.name());
+    }
+
+    public WorkFlowEventType getWorkflowEventType() {
+        return eventType;
+    }
+
+    public long getPackageId() {
+        return packageId;
     }
 
     @Override
     public String toString() {
         return "WorkflowEvent{" +
                 "siteId='" + siteId + '\'' +
                 ", timestamp=" + timestamp +
-                ", user=" + user +
```

Comment:
Why remove user?

Reply:
I believe this is an error. It was removed because it was not used. I will add it back

Reply:
Done